scripts/e2e.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+PODMAN="podman"
+PSQL_CONTAINER="sippy-e2e-test-postgresql"
+PSQL_PORT="23433"
+
+clean_up () {
+    ARG=$?
+    echo "Killing sippy API child process: $CHILD_PID"
+	kill $CHILD_PID
+	echo "Tearing down container $PSQL_CONTAINER"
+	$PODMAN stop -i $PSQL_CONTAINER
+	$PODMAN rm -i $PSQL_CONTAINER
+    exit $ARG
+}
+trap clean_up EXIT
+
+# make sure no old container running
+echo "Cleaning up old sippy postgresql container if present"
+$PODMAN stop -i $PSQL_CONTAINER
+$PODMAN rm -i $PSQL_CONTAINER
+
+# start postgresql in a container:
+echo "Starting new sippy postgresql container: $PSQL_CONTAINER"
+$PODMAN run --name $PSQL_CONTAINER -e POSTGRES_PASSWORD=password -p $PSQL_PORT:5432 -d quay.io/enterprisedb/postgresql
+
+echo "Wait 5s for postgresql to start..."
+sleep 5
+
+echo "Loading database..."
+# use an old release here as they have very few job runs and thus import quickly, ~5 minutes
+#make build
+#./sippy --load-database \
+#  --log-level=debug \
+#  --load-prow=true \
+#  --load-testgrid=false \
+#  --release 4.7 \
+#  --database-dsn="postgresql://postgres:password@localhost:$PSQL_PORT/postgres" \
+#  --mode=ocp \
+#  --config ./config/openshift.yaml \
+#  --google-service-account-credential-file ~/creds/openshift-ci-data-analysis-dgoodwin-gcs-3a2d16e2ee78.json
```

Comment:
Anyone know how I can get such a GCS cred when this runs in a CI job?

Reply:
https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/ has details, if you make it a CI step it's pretty easy